packages/cli/src/auth.ts

```diff
@@ -97,3 +111,64 @@ function printMe(medplum: MedplumClient): void {
     console.log('Not logged in');
   }
 }
+
+async function medplumAuthorizationCodeLogin(medplum: MedplumClient): Promise<void> {
+  await startWebServer(medplum);
+  const loginUrl = new URL(medplum.getAuthorizeUrl());
+  loginUrl.searchParams.set('client_id', clientId);
+  loginUrl.searchParams.set('redirect_uri', redirectUri);
+  loginUrl.searchParams.set('scope', 'openid');
+  loginUrl.searchParams.set('response_type', 'code');
+  loginUrl.searchParams.set('prompt', 'login');
+  await openBrowser(loginUrl.toString());
+}
+
+function isThereExistingProfileName(profileName?: string): boolean {
```

Comment:
Let's use `profileExists` rather than `isThereExistingProfileName`

How is that different than `checkForProfile` in `packages\cli\src\util\client.ts`?

When creating util methods, we should make sure they are tight, single purpose, and clearly named.  We've been evolving these utils, and it may make sense to step back and come up with a clear set of helpers:

* `loadProfile` or `readProfile`
* `saveProfile` or `writeProfile`
* `profileExists`

It looks like you cleaned up most cases where we're calling `new FileSystemStorage(profileName)` (good!), but maybe not all?